markdown/events/2022/bytesize-gitpod.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Bytesize: gitpod.io'
+subtitle: Chris Wyatt - Seqera Labs
+type: talk
+start_date: '2022-06-14'
+start_time: '13:00 CET'
+end_date: '2022-06-14'
+end_time: '13:30 CET'
+location_url:
+  - https://kth-se.zoom.us/j/68390542812
+---
+
+# nf-core/bytesize
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: gitpod.io
+
+This week, Chris Wyatt ([@chriswyatt1](https://github.com/chriswyatt1)) will show us the functionality that comes with the addition of and how to use the .gitpod.yml as part of the pipeline template.
```

Comment:
```suggestion
This week, Chris Wyatt ([@chriswyatt1](https://github.com/chriswyatt1)) will show us how to use GitPod (<https://gitpod.io>) for testing, development and training.  GitPod gives you a complete cloud-based environment to work in for free, with VSCode editor integration (in your browser or locally).

Chris will show us how it works, how you can use the environments on offer and how to use the `.gitpod.yml` configuration that now comes as part of the nf-core pipeline template.
```

This week, Chris Wyatt ([@chriswyatt1](https://github.com/chriswyatt1)) will show us how to use GitPod (<https://gitpod.io>) for testing, development and training.  GitPod gives you a complete cloud-based environment to work in for free, with VSCode editor integration (in your browser or locally).

Chris will show us how it works, how you can use the environments on offer and how to use the `.gitpod.yml` configuration that now comes as part of the nf-core pipeline template.